inception-project/inception
# #2443 Keep Order of Search Results

**What's in the PR**
- The sortLeaves method in MtasDocumentIndexTest
- In essence, this method takes care that the search order of the documents is the same as the document id.
- This leads to a stable ordering of the search results. 
- Example 1: The search results ordering is kept even if lowLevelPaging is active.
- Example 2: The search results are ordered according to document id and the currently displayed one does not come last 

**How to test manually**
* Play around with the search (e.g. (de-) activate lowLevelPaging, make the same search again after annotating some document, ...) and observe that the search results are always ordered according to document id.
* Of course, this is not valid if some other kind of grouping is active. 

**Automatic testing**

**Documentation**
